cinema/serializers.py

```diff
@@ -68,13 +78,60 @@ class Meta:
             "movie_title",
             "cinema_hall_name",
             "cinema_hall_capacity",
+            "tickets_available",
         )
 
 
+class TicketSerializer(serializers.ModelSerializer):
+    def validate(self, attrs):
+        data = super(TicketSerializer, self).validate(attrs=attrs)
+        Ticket.validate_ticket(
+            attrs["row"], attrs["seat"], attrs["movie_session"]
+        )
```

Comment:
The `validate_ticket` method is called on the `Ticket` model, but it's not clear if this method exists. Ensure that the `Ticket` model has a `validate_ticket` method implemented to avoid runtime errors.